pkg/agent/plugin/workloadattestor/docker/docker_test.go

```diff
@@ -200,6 +203,42 @@ func TestDockerError(t *testing.T) {
 	require.Nil(t, res)
 }
 
+func TestDockerErrorRetries(t *testing.T) {
```

Comment:
Can you add a test to cover the canceled context path?

I really wish there was inline coverage reporting. I don't know if we can get that on github.